database/src/main/java/com/firebase/ui/database/FirebaseIndexListAdapter.java

```diff
@@ -31,9 +31,11 @@
  *     listView.setListAdapter(adapter);
  * </pre>
  *
- * @param <T> The class type to use as a model for the data
- *           contained in the children of the given Firebase location
+ * @param <T> The class type to use as a model for the data contained in the children of the given
+ *            Firebase location
+ * @deprecated use {@link com.firebase.ui.database.adapter.FirebaseIndexRecyclerAdapter} instead
  */
+@Deprecated
```

Comment:
Ok this is the biggest thing we need to talk about: the change in package and the mass-deprecation.  This is something we should avoid at nearly all costs.  A change like this would need to be in version `2.0` for sure, as even if SemVer does not have a strong opinion on deprecations this is essentially asking every FBUI developer to change all of their code.